api/ownership.go

```diff
@@ -189,8 +198,13 @@ func (o *Ownership) Update(newownerInfo *Ownership, user *auth.UserInfo) error {
 		o.Acls = newownerInfo.GetAcls()
 
 		// Only the admin can change the owner
-		if o.IsAdminByUser(user) && len(newownerInfo.Owner) != 0 {
-			o.Owner = newownerInfo.Owner
+		if len(newownerInfo.Owner) != 0 {
```

Comment:
Could just use `o.HasAnOwner()` here to be more clear

Reply:
I like that, good idea. I'll make the change 👍